src/main/java/org/jboss/modules/xml/SystemPropertyConditionBuilder.java

```diff
@@ -1,3 +1,21 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Fixing white space is okay but if you want to make things consistent, we should be removing the header rather than adding it. It doesn't perform any legal purpose.

Reply:
I thought every source file should have a license header @dmlloyd .

Reply:
It is not necessary. For new projects I am not including them at all, unless the source code came from somewhere else.

Reply:
So you propose to remove added license headers, right?

Reply:
Correct.

Reply:
Done